pkg/mkverification-raw-efi/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# This mkverification-raw-efi produces the raw EFI partition for EVE,
+# including the files in efi-files in the image.  This includes:
+#
+#   /EFI/BOOT/grub.cfg - Chainloads main bootloader
+#   /UsbInvocationScript.txt - Enables USB boot on Dell 3000 series
+#
+FROM lfedge/eve-alpine:145f062a40639b6c65efa36bed1c5614b873be52 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+ENV BUILD_PKGS mkinitfs grep patch git make coreutils musl-dev gcc g++ perl autoconf automake libtool file bsd-compat-headers libc-dev util-linux-dev flex linux-headers glib-dev libxrandr-dev zlib-dev libusb-dev kmod-dev curl
+ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux squashfs-tools coreutils tar dmidecode smartmontools libaio libaio-dev perl glib zlib libusb curl xz pciutils usbutils hdparm util-linux wireless-tools libxrandr eudev-libs libusb sudo
+RUN eve-alpine-deploy.sh
+
+RUN echo -e "http://dl-cdn.alpinelinux.org/alpine/v3.16/main\nhttp://dl-cdn.alpinelinux.org/alpine/v3.16/community\nhttp://dl-cdn.alpinelinux.org/alpine/v3.16/test" > /etc/apk/repositories
+RUN apk add -p /out fio iperf3 sysstat lm_sensors acpi iw libdrm hwinfo dhclient
```

Comment:
Why are you using apk here like this? You should be able to list all your dependencies in *PKG* variables.... and not require network access during the building....

Reply:
These packages are not present in our cache (checkout pkg/alpine/mirrors).